Sources/Shared/Operators/Operators.swift

```diff
@@ -32,3 +33,7 @@ public func <+ <T>(inout left: [T], right: T?) {
   guard let right = right else { return }
   left.append(right)
 }
+
+public prefix func <? <T: DefaultType>(value: T?) -> T {
+  return value ?? T.defaultValue
+}
```

Comment:
@zenangst How do you like it to be? `<-` is infix, but it requires that the property of the struct be initialized beforehand 😬